src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/AddDraftApprenticeship.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@model SFA.DAS.ProviderCommitments.Web.Models.AddDraftApprenticeshipViewModel
+
+@{
+    ViewData["Title"] = "Apprenticeships";
+    ViewBag.GaData.Vpv = "/unapproved/apprentices/add-apprentice";
```

Comment:
Maybe this should be `/unapproved/apprentices/add'? Then again, I wondered if the word "cohort" should appear in the vpv, because we're actually adding an apprentice to a cohort. Eg. `/unapproved/cohort/apprentices/add'. Hmm, I'm not sure though. Reservations don't seem to have done that on their side. But it feels strange not to mention cohort. Happy to discuss.

Reply:
updated